docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,57 @@
+    * We also have a supplementary application called :ref:`Vertiq Testing Tool <vertiq_testing_tool_guide>` to help you test your module. You can download and run Vertiq Testing Tool for more advanced testing of your module's configuration.
+
+#. Why does my module not spin when I send it a PWM signal?
+    * Vertiq modules require initial configuration before being able to spin with PWM commands. You can configure your motor using :ref:`Control Center <control_center_start_guide>`. The most commonly missed configuration when trying to spin with PWM is the :ref:`Motor Direction <throttle_direction>`, which must be set to something other than unconfigured. We recommend setting it to 2D Counter-Clockwise for your initial setup. You should also check what :ref:`Mode <throttle_mode>` is set on your module and that the corresponding :ref:`Maximum <throttle_maximums>`  for that mode is set. If you are still having trouble, refer to the :ref:`PWM and DSHOT tutorial <hobby_fc_tutorial>` or your module's Getting Started documentation: :ref:`Speed <speed_module_start_guide>`, :ref:`Servo <servo_module_start_guide>`, or :ref:`Pulsing <pulsing_module_start_guide>`.
+
+#. I’m trying to use DroneCAN but my motor isn’t spinning.
+    * A common issue that may cause your module not to spin is that its parameters are not configured correctly. Please confirm that you have properly configured the :ref:`Motor Direction <throttle_direction>`, :ref:`Mode <throttle_mode>`, and :ref:`Maximums <throttle_maximums>` for your module. Also, confirm that your configured :ref:`ESC Index <dronecan_px4_tutorial_esc_index>` is correct for the module that you are trying to control. If your module is not using :ref:`Arming Bypass <dronecan_arming_and_bypass>`, confirm that you are sending DroneCAN commands that will :ref:`arm <dronecan_arming_and_bypass>` the module initially. By default, commands close to 0% throttle should arm the module. For more information on how to use DroneCAN with your module, please refer to our :ref:`DroneCan Integration with PX4 and Ardupilot <dronecan_fc_tutorial>` documentation.
+
+#. What hardware do I need prior to receiving my module?
```

Comment:
should this move to be #1?

Reply:
Initially I had it with the first FAQ question but since it's a DroneCAN specific question I think it's better on its own.

Reply:
@Vertiq-Ben It's DroneCAN specific? Isn't this the question about what hardware is needed?

Reply:
Whoops I thought you were talking about the question before on line 18. I will move "What hardware do I need prior to receiving my module" as the first FAQ.